src/track.h

```diff
@@ -29,15 +29,24 @@
 
 extern u8 n_rollovers;
 
+#define SENTINEL_WORDS_EACH 1
```

Comment:
Sentinel is for memory corruption testing? Add some comments about it? Do you think we should keep it in here now that the issue has been ostensibly resolved?